src/advert/adservers.txt

```diff
@@ -48,6 +48,7 @@
 ||filebanner.com^$third-party
 ||forumiklan.com^$third-party
 ||goingsilltrusty.com^
+||googleusercontent.com^
```

Comment:
This domain serves general images, not ad specific, if you want to block it must be specific

Reply:
Yes the image hovering on top of the page comes from it so I assumed  its the problem. But thanks for letting me know. 
To make it specific 
`15.235.11.45,oploverz.co.in##a[title*="388hero slot gacor"]`
would do the same but through Element Hiding